docs/resource-policy/setup.md

```diff
@@ -0,0 +1,61 @@
+# Setup and Usage
+
+When you want to try NRI Resource Policy, here is the list of things
+you need to do, assuming you already have a Kubernetes\* cluster up and
+running, using either `containerd` or `cri-o` as the runtime.
+
+  * [Install](installation.md) NRI Resource Policy DaemonSet deployment file.
+  * Runtime (containerd / cri-o) configuration
+
+For NRI Resource Policy, you need to provide a configuration file. The default
+configuration ConfigMap file can be found in the DaemonSet deployment yaml file.
+You can edit it as needed.
```

Comment:
I think this info should be part of the "Installation" doc, if not already covered there